js/components/frame.js

```diff
@@ -965,6 +965,11 @@ class Frame extends ImmutableComponent {
         }))
       }
     })
+    this.webview.addEventListener('did-get-response-details', (details) => {
+      if (getSetting(settings.PAYMENTS_ENABLED) && details.originalURL === this.props.location) {
```

Comment:
NOTE: this event will only get sent if payments is enabled (to prevent unneeded overhead)